0/a-burke/molecule.py

```diff
@@ -0,0 +1,85 @@
+#Author: Ally Burke
+
+
+import sys
+sys.path.append('../../extra-files')
+import copy
+import numpy as np
+from masses import get_charge
+from masses import get_mass
+
+class Molecule(object):
+
+	#Used to initialize an instance of the Molecule class
+
+	def __init__(self, geom_str, units = "Angstroms"):
+		self.units = units				#Specifies units for spatial coordinates
+		self.read(geom_str)				#Reads in the geometry file for the specific molecule	
+		self.natom = self.read(geom_str)		#Specifies the number of atoms in the molecule
+		self.labels = self.read(geom_str)		#Specifies list of atom symbols following the input file	
+		
+		#unsure how to reference parameters without hardcoding
```

Comment:
I recommend you set up the masses and charges in the read function. You can initialize them here (e.g. self.masses = []), but it will be simplest to actually define most of variable inside the read function. I recommend defining the "labels" variable just as you have it on line 37, then pass in the label as the argument for the get_mass or get_charge functions.